forte/data/base_pack.py

```diff
@@ -468,22 +468,35 @@ def get(
         """
         raise NotImplementedError
 
-    def get_single(self, entry_type: Union[str, Type[EntryType]]) -> EntryType:
+    def get_single(
```

Comment:
We can revert the changes here since `get_single` is no longer used in the current implementation.

Reply:
I think there are still many test cases dependent on this function. Even if it's no longer needed, maybe we can raise an issue for it and fix it in another PR.

Reply:
no, not deleting `get_single`, but revert it to the previous state. This function is still needed.